packages/itwin/tree-widget/src/components/trees/stateless/common/components/VisibilityTree.tsx

```diff
@@ -135,11 +149,21 @@ function VisibilityTreeImpl({
           <VisibilityTreeRenderer
             rootNodes={rootNodes}
             {...treeProps}
+            expandNode={(node, isExpanded) => {
```

Comment:
Looks like `expandNode`, `onFilterClick` and `onCheckboxClicked` should be memoized as they are used as dependencies in `useEffect` inside `VisibilityTreeRenderer`

Reply:
Good catch, memoized the callbacks.